syntax = "proto2";

enum __message_id
{
	test_message_id = 0x100;
}

message test_message
{
	required int32 value1 = 1;
	required uint64 value2 = 2;
}

